### PR TITLE
More dependency updates

### PR DESCRIPTION
**Pull request by jmartin-sul**
* moved non-Stanford/Hydra gem inclusions to correct section in Gemfile
* bundle update

### CODE REVIEW

**Comment by jmartin-sul** [comment]
IIRC from standup yesterday, @tingulfsen said he'd be releasing a new version of modsulator.  maybe this should wait on that, and then incorporate it too?

**Comment by atz** [comment]
No need to wait, AFIAC.  Do need to rebase though.

**Comment by jmartin-sul** [comment]
rebased, ready for review.  @tingulfsen says he'll deal w/ modsulator stuff tomorrow morning (and the dependency is indirect anyway).

File: Gemfile [hunk 1]
```diff
@@ -45,13 +48,10 @@ gem 'is_it_working-cbeer'
...
 gem 'mods_display'
-gem 'rails', '~> 4.0' # specifying because we expect a major vers upgrade to break things
 gem 'responders', '~> 2.0'
 gem 'rsolr'
 gem 'rsolr-client-cert', '~> 0.5.2'
-gem 'ruby-prof'
 gem 'solrizer'
-gem 'sprockets', '~> 3.4'
 gem 'stanford-mods'
 gem 'sul_styles', '~> 0.3.0' # later versions require ruby 2.1 or higher
```
Comment (darrenleeweber) [inline Gemfile]: Anything really different here?  Just putting the deck into order.

**Pull request merged**